### ISSUE TITLE
push and pull classes for the grids

### ISSUE THREAD

**Issue by gmclelland** [stubbornella/oocss#75]
Is there any we could get push and pull classes for the grids to help control the visual order when the source order is different?  This would be helpful when using media queries.  What are your thoughts?

**Comment by stubbornella** [comment]
Can you explain your use case a little bit more?

Nicole

On Jun 7, 2011, at 8:30 AM, gmclelland wrote:

> Is there any we could get push and pull classes for the grids to help control the visual order when the source order is different?  This would be helpful when using media queries.  What are your thoughts?
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/stubbornella/oocss/issues/75

## 

Nicole Sullivan

Object Oriented CSS  http://wiki.github.com/stubbornella/oocss
Smush it  http://smush.it
Website  http://stubbornella.org
Tweet!  http://twitter.com/stubbornella
Book  Even Faster Websites with O'Reilly

**Comment by gmclelland** [comment]
When you use just grids for laying out your website you really don't have any control over the how the layout collapses when using media queries.  For example say you have a standard three column website using oocss grids.  When viewing it on phone the left column will come first, then the middle, then the right column.  Instead, I would like the middle column to come first and then the left and right column all stacked on top of each other.

Basically I would like to use oocss grids and have a site that collapses down like the sites on http://mediaqueri.es/ .  It would be awesome if you could throw up an example of a responsive layout that uses oocss.

The 960.gs gives the push and pull classes for each grid unit so that you can visually reorder the columns.
